### DISCUSSION TITLE
`<rh-cta>` or `<rh-tile>` variant for learning paths

### DISCUSSION THREAD

**Discussion by marionnegp** [RedHat-UX/red-hat-design-system#1716] [Ideas]
Learning paths on cloud.redhat.com and access.redhat.com are using a type of CTA that looks like these examples:

**[Example](https://access.redhat.com/start) from access.redhat.com/start homepage**
![Screenshot 2024-07-16 at 12 15 22 PM](https://github.com/user-attachments/assets/65690b1d-ed4a-4ac0-a66c-a7c668e80c5a)

**[Example](https://access.redhat.com/start/learn:get-set-red-hat/resource/resources:manage-your-red-hat-account-and-preferences) from access.redhat.com/start learning path**
![Screenshot 2024-07-16 at 12 14 52 PM](https://github.com/user-attachments/assets/d2bcbec3-ed88-4e0e-b604-1f03bfe3e969)

**[Example](https://cloud.redhat.com/learning/learn:getting-started-red-hat-openshift-service-aws-rosa/resource/resources:requirements-using-red-hat-openshift-service-aws-rosa) from cloud.redhat.com**
![Screenshot 2024-07-16 at 12 15 47 PM](https://github.com/user-attachments/assets/9181d212-187c-4ccd-9d50-e917847a3ac9)

Most of the use cases blend the function of a CTA and pagination. Is this something that we should add to RHDS as either a variant of `<rh-cta>` or as a new element?

**Comment by bennypowers** [comment]
@adamjohnson is that too much in a link?

**Reply by adamjohnson** [reply]
Bootstrap has the concept of a [Stretched Link](https://getbootstrap.com/docs/5.3/helpers/stretched-link/). That's the proper way to achieve this functionality. Minimal link text, maximum link target.

**Reply by bennypowers** [reply]
So is this a tile with header and footer but no body?

**Reply by adamjohnson** [reply]
That sounds about right. Plus:

  * The ability to have the whole tile clickable
  * Proper font size for the footer (or body) text

**Reply by bennypowers** [reply]
iirc "whole tile clickable" is the essential definition of tile

**Comment by coreyvickery** [comment]
@marionnegp Could we use `<rh-tile>`?

**Reply by marionnegp** [reply]
Yeah, I think that's a possibility. @markcaron wanted to cover this if we have time in office hours.
